src/App.vue

```diff
@@ -68,15 +63,3 @@ watch(route, () => {
     <div id="modal" />
   </div>
 </template>
-
-<style>
-.backdrop {
-  position: fixed;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  z-index: 99;
-  background: rgba(0, 0, 0, 0.4);
-}
-</style>
```

Comment:
Why was this removed? Without this we won't have backdrop when opening sidebar. Is this intentional?

Reply:
Good catch, I was assuming it was a duplicate code with the modal.